content/03-types/03-document/01-what-are-document-dbs.mdx

```diff
@@ -0,0 +1,107 @@
+
+<PrismaOutlinks>
+
+Prisma's [MongoDB connector](https://www.prisma.io/docs/concepts/database-connectors/mongodb) has recently been [promoted to general availability](https://www.prisma.io/blog/mongodb-general-availability-pixnun6mffmu)!  With this change, you can use the [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) to manage production MongoDB databases with confidence.
+
+Join us in celebrating this milestone on April 25-29 with our [virtual MongoDB Launch week](https://www.prisma.io/mongodblaunch).  There will be exclusive workshops, opportunities for Atlas credits, great swag, and much more!
+
+</PrismaOutlinks>
+
+## Documents 
+As discussed, documents are at the crux of any document database. Depending on the document database, documents encapsulate and encode data in JSON, XML, YAML, or a binary form like BSON.
+
+One of the attractive elements for developers to use the document model is its similarities to objects in programming languages. There is a familiarity with the structure or lack thereof when using documents. 
+
+The basic format of a document looks as follows:
+
+```
+	 {
+	   field1: value1,
+	   field2: value2,
+	   field3: value3,
+	   ...
+	   fieldN: valueN
+	}
+```
+
+Expanding on the basic syntax, a single document within a collection of authors could look like this:
+
+```
+	{
+		"ID": "001",
+		"Books": { 'Grey Bees', 'Death and the Penguin' },
+		"Author": "Andrey Kurkov"
+	}
+```    
+
+What is critical to notice is the ability to store multiple books within the `Books` field. In a relational database, this would not be possible. There would need to be an `Author` table and a `Book` table joined by a [key](/intro/database-glossary#foreign-key). This foreign key would most likely be something like `author.id` where every record in the `Book` table. We can visualize the difference in the following tables:
+
+| author.id | name  |
+|----------- |---|
+|001      | Andrey Kurkov |
+
+| book.title | author.id  |
+|----------- |---|
+|Grey Bees      | 001 |
+|Death and the Penguin      | 001 |
+
+With an idea of the structure and capabilities of documents, we can take a step further and explore the advantages and disadvantages that the document model presents.
+
+## Advantages of the document model
+There are clear strengths and weaknesses with a document database, and it depends from application to application whether the document model is the right fit. The document model's flexibility, ease of scale, and quick-start agility are advantages of the document model with considerable trade-offs.
+
+### Flexibility
+Document databases offer flexibility unmatched by relational databases. 
```

Comment:
Just noticed the single line break here so I wanted to point it out.  I think it works fine as either a single paragraph (how it's rendering now) or separate paragraphs, so just wanted to make sure it's doing what you wanted it to do :+1: